### PR TITLE
chore: add empty element templates directory

### PR DESCRIPTION
**Pull request by philippfromme**
Based on feedback from @GeethaParthasarathy I've added an `element-templates` directory to avoid confusion.

### CODE REVIEW

**Review by barmac — APPROVED** [review]
Makes sense.

File: resources/platform/base/resources/element-templates/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Element Templates Directory
+
+Place element templates into this directory to use them in the Camunda Modeler.
```
Comment (nikku) [inline resources/platform/base/resources/element-templates/README.md:3]: Proposal: Document additional, local discovery mechanism, too.

Essentially we have: 

* Put templates in modeler directory
* Put templates in local user configuration
* Put templates local to your diagrams (in a `.camunda/element-templates` folder) :arrow_left: recommended!

Comment (nikku) [reply resources/platform/base/resources/element-templates/README.md:3]: I also believe that a more appropriate link in this documentation is https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/configuring-templates/. It captures all options.

Comment (philippfromme) [reply resources/platform/base/resources/element-templates/README.md:3]: Will follow up.

**Pull request merged**